arch/arm/core/cortex_m/Kconfig

```diff
@@ -103,6 +103,14 @@ config ISA_THUMB2
 	  technology is featured in the  processor, and in all ARMv7
 	  architecture-based processors.
 
+config CPU_CORTEX_M_HAS_ISB
```

Comment:
This is the wrong feature name, M0 & M0+ have ISB, the issue is that M0 doesn't support unpriv mode, thus the code doesn't need to exist.  The priv/unpriv mode is an optional feature on M0+ (so we need to review the M0+ SoC to see if they implement the feature)